app/models/mappers/check-activities.js

```diff
@@ -388,10 +388,19 @@ const mapCreatedEventToCheckActivityRows = (event) => {
 const mapImportEventToCheckActivityRows = (event) => {
   const dateAndTeamMemberData = getDateAndTeamMemberFromEvent(event)
 
-  return [{
-    activityLabel: `Comments made by index users: ${event.added?.comment}`,
-    ...dateAndTeamMemberData
-  }]
+  if (event.type === 'uk.gov.defra.ddi.event.import') {
+    return [{
+      activityLabel: `Comments made by index users: ${event.added?.comment}`,
```

Comment:
two small corrections...would it be ok to do:

``[
        {``

and there's a little coverage missing on it.  Being really fussy here, but we managed to get 100% would be great to keep it.

Reply:
edited: forget the first correction, just coverage